src/cosalib/ibmcloud.py

```diff
@@ -38,9 +38,7 @@
         "image_suffix": "ova.gz",
         "platform": "powervs",
         "compression": "gzip",
-        "tar_members": [
-            "disk.raw"
-        ]
```

Comment:
Maybe let's add a comment here that the disk image is added as part of `write_ova()`.

Reply:
Added the comment.